packages/flame/lib/src/components/text_component.dart

```diff
@@ -46,6 +46,6 @@ class TextComponent extends PositionComponent {
   @override
   void render(Canvas canvas) {
     super.render(canvas);
-    _textRenderer.render(canvas, text, Vector2.zero());
+    _textRenderer.render(canvas, text, Vector2.zero(), anchor: anchor);
```

Comment:
I think it would be cool if we could add a better example of texts, using dashbook properties so we could easily test the variations of the properties that texts have.

Could be a follow up if you don't want to do on this PR.

Reply:
Sounds great, but let's do it as a follow-up. :)